src/core/Settings.js

```diff
@@ -1299,7 +1299,12 @@ function Settings() {
                 rtpSafetyFactor: 5,
                 mode: Constants.CMCD_MODE_QUERY,
                 enabledKeys: Constants.CMCD_AVAILABLE_KEYS,
-                includeInRequests: ['segment']
+                includeInRequests: ['segment'],
+                reporting: {
+                    requestMode: {
+                        version: null
```

Comment:
I think that we should use `version: 1` instead of `version: null`. Defining a variable as null is unnecessary.